src/installer/corehost/hostfxr_iface.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __HOSTFXR_IFACE_H__
+#define __HOSTFXR_IFACE_H__
+
+#include "hostfxr.h"
+#include "pal.h"
+#include "error_codes.h"
+
+class hostfxr
```

Comment:
The class name must match the file name. So, the class should be called hostfxr_iface_t.
However, in keeping with the hostpolicy_resolver naming, this class (and file) should probably be called hostfxr_resolver_t.

Reply:
Forgot about this. Why the "_t" part? 
Could the type and file be both just `hostfxr_resolver` ?

Reply:
Every type in the host ends with `_t`. While I don't particularly like it, we should follow the convention.